components/MarkdownEditor.tsx

```diff
@@ -34,7 +33,6 @@ const MarkdownEditor = ({
       }}
       handleDOMEvents={{
```

Comment:
This does nothing now, should be removed?

Reply:
Yep, remove this and the onSave argument to the component. Nice find

Reply:
Done